src/Features/Core/Portable/Diagnostics/Analyzers/SimplifyTypeNamesDiagnosticAnalyzerBase.cs

```diff
@@ -74,17 +73,6 @@ protected SimplifyTypeNamesDiagnosticAnalyzerBase(ImmutableArray<TLanguageKindEn
             _kindsOfInterest = kindsOfInterest;
         }
 
-        public bool OpenFileOnly(Workspace workspace)
-        {
-            var preferTypeKeywordInDeclarationOption = workspace.Options.GetOption(
```

Comment:
same here. not sure what the reason was that IDE refuse to use regular diagnostic system but use this system. @mavasani do you remember? was it due to naming convention analyzer? which they say they don't know what diagnostic they have either since user can configure those?